src/POData/UriProcessor/QueryProcessor/QueryProcessor.php

```diff
@@ -222,7 +222,7 @@ private function processOrderBy()
             $orderBy = rtrim($orderBy, ', ');
         }
 
-        if (!is_null($orderBy)) {
+        if (!is_null($orderBy) && '' != trim($orderBy)) {
```

Comment:
Prevent an empty string reaching the check in the OrderByParser - two lines of defence against the barfage